src/main/java/org/opensearch/sdk/ClusterSettingResponseHandler.java

```diff
@@ -0,0 +1,40 @@
+package org.opensearch.sdk;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.message.ParameterizedMessage;
+import org.opensearch.cluster.ClusterSettingsResponse;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.threadpool.ThreadPool;
+import org.opensearch.transport.TransportException;
+import org.opensearch.transport.TransportResponseHandler;
+
+import java.io.IOException;
+import java.util.concurrent.CountDownLatch;
+
+public class ClusterSettingResponseHandler implements TransportResponseHandler<ClusterSettingsResponse> {
+    private static final Logger logger = LogManager.getLogger(ClusterSettingResponseHandler.class);
+    final CountDownLatch inProgressLatch = new CountDownLatch(1);
+
+    @Override
+    public void handleResponse(ClusterSettingsResponse response) {
+        logger.info("received {}", response);
+        inProgressLatch.countDown();
+    }
+
+    @Override
+    public void handleException(TransportException exp) {
+        logger.debug(new ParameterizedMessage("ClusterSettingRequest failed"), exp);
```

Comment:
Why wrap the String in a ParameterizedMessage instead of [passing directly](https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/Logger.html#debug-java.lang.String-java.lang.Object...-)?